roles/flag-lifecycle/per-environment.tf

```diff
@@ -151,17 +254,17 @@ resource "launchdarkly_custom_role" "apply-changes" {
 
 
 
-resource "launchdarkly_custom_role" "trigger-manager" {
```

Comment:
I think we should leave trigger manager even if we don't go over assigning. It makes it so they can easily use triggers when they are ready to use them in the future without having to go back and create a new role

Reply:
Alternatively, we can separate it into an auxiliary module since we don't cover it in platform readiness. 

Or we can add a note like: "We also created these roles for you to use later"